### ISSUE TITLE
User Cannot Update/Create Call With Keyboard Input

### ISSUE THREAD

**Issue by Scott-James-Hurley** [UserOfficeProject/user-office-project-issue-tracker#1685]
A User cannot access all the fields in the create/update call modal with just a keyboard. Trying to tab between the fields results in only the first few being selectable. A keyboard user should be able to access all fields and controls using just a keyboard.
